### ISSUE TITLE
Alexa - recognition issues? 

### ISSUE THREAD

**Issue by theroninhunter** [stevenleeg/geemusic#281]
So I've got it all setup, testing commands worked etc in the dev console. 

But if I say "Alexa ask geemusic, start playlist My music" it starts a random station instead

**Comment by Cralex** [comment]
It always thinks I'm talking about some dumb skill from the skills store when I try to invoke it. So, I tried changing the invocation to something else and now Alexa recognizes it. I changed it to gear music.

...Now I just need to get it to play music. It’s getting invalid responses or something. (Heroku hosted)

**Comment by fergyfresh** [comment]
So the feature that I use all the time is `Alexa ask gee music to play some music` which plays the I'm feeling lucky playlist. I wrote that feature ;).

Now @theroninhunter the playlist feature DEFINITELY doesn't work. I recently made a playlist and tried to get it to work and I found the same issues as you. I also tried to add to a playlist with the `gmusicapi` a few weeks ago and that wasn't really working either. It could be an API change on google's end which is causing us to not find any playlists or not be able to play any playlists. It could also very much be this repo's fault too as we have no tests for testing playlists.

As far as you @Cralex I don't konw what you're issue is and you will have to be more specific and provide logs for me to discern what is happening on your end.
